package.json

```diff
@@ -45,7 +45,7 @@
     "publish-beta": "yarn bootstrap && yarn test && lerna publish --npm-tag beta --cd-version prerelease",
     "test": "scripts/test.sh",
     "test-ci": "scripts/test.sh cover && scripts/test.sh bench",
-    "test-browser": "webpack-dev-server --config test/webpack.config.js --env.test --progress --hot --open",
+    "test-browser": "node test/start.js browser",
```

Comment:
Does the browser auto close, or will it still stay open for debugging? Pretty important to me, just had a problem with this in `math.gl/bench`...

Reply:
It stays open.